src/test/kotlin/com/okp4/processor/cosmos/TopologyTest.kt

```diff
@@ -45,8 +45,12 @@ class TopologyTest : BehaviorSpec({
     ).toProperties()
 
     given("A topology") {
-        val topology = topology(config)
-        val testDriver = TopologyTestDriver(topology, config)
+        val topologyProducer = TopologyProducer()
+        topologyProducer.topicIn = "in"
+        topologyProducer.topicOut = "out"
+        topologyProducer.topicError = "error"
```

Comment:
Just to have something to say! 😁

```suggestion
        val topologyProducer = TopologyProducer().apply {
            topicIn = "in"
            topicOut = "out"
            topicError = "error"
        }
```